pages/open_cas_linux/guide_product_specs_requirements.md

```diff
@@ -0,0 +1,95 @@
+| CPU Overhead     | Open CAS Linux consumes minimal CPU bandwidth (less than 10% in most cases). Ensure that Open CAS Linux has adequate CPU capacity to function optimally.                                                                                                                                                                                          |
+| Flash/SSD        | Any Linux flash device (SAS, SATA, PCIe\*, Fibre Channel, RAID) is supported, and can be direct attached, expander attached, or attached via SAN (with a single worker). Additionally, the following Intel® SSDs have been fully qualified:                                                                                              |
+| Storage          | Primary storage devices: Any device that appears as a local block device. For example, local disk, RAID, SAN iSCSI (via Fibre Channel, Infiniband, Ethernet), etc. Core device (HDD) logical block size must be 512 bytes or larger. Operating system must be on separate partition or drive than the core storage device to be cached. |
+| File Systems     | The following file systems are supported for primary storage or core storage:                                                                                                                                                                                                                                                           |
+| Software         | The following prerequisite software must be installed prior to Open CAS Linux installation:                                                                                                                                                                                                                                                  |
+| Power Management | Power Management Suspend (S3) and Hibernate (S4) states must be disabled.                                                                                                                                                                                                                                                               |
+
+-   Intel® SSD DC S3700 Series
+
+-   Intel® SSD DC P3700 Series
+
+Cache device logical block size must be smaller than or equal to the logical
+block size of the core storage device. 60GB or larger capacity is recommended.
+
+**NOTE:** Older firmware on some SSDs lose their format after reboot and cause
+data loss. We highly recommend that you upgrade to the latest recommended
+firmware and boot loader versions for your devices. For example, as of the
+writing of this document, the firmware versions (8DV101H0 for firmware &
```

Comment:
Is this up-to-date? I think this section should be removed (all mentions of Intel SSDs in this chapter).

Reply:
I will fix this